src/app/lib/utilities/ichefURL/index.js

```diff
@@ -15,4 +15,16 @@ const getPlaceholderSrc = src => {
   return newUrl.join('/');
 };
 
-export default getPlaceholderSrc;
+const getIChefURL = ({ originCode, locator, resolution }) => {
+  if (originCode === 'pips') {
+    return locator;
+  }
+
+  if (originCode === 'mpv') {
+    return getPlaceholderSrc(locator);
```

Comment:
Just curious as to whether this method name should be changed if it is doing more than just getting a placeholder src?

Reply:
I agree, the function does more than just returning a placeholder src.